themes/bootstrap3/templates/search/searchbox.phtml

```diff
@@ -1,12 +1,27 @@
 <?php
     // Set default value if necessary:
-    if (!isset($this->searchClassId)) {
-        $config = $this->config()->get('config');
-        $this->searchClassId = $config->Site->defaultSearchBackend ?? 'Solr';
+    $options = null;
```

Comment:
Since we have an if..else, I don't think it makes sense to initialize these variables before the if -- some are already redefined by both branches, rendering the initialization redundant, and the remainder could just be moved into the else clause.

Reply:
Agreed. Now cleaned up.